### PR TITLE
Kamu UI graph improvement   details side panel

### PR DESCRIPTION
**Pull request by dmitriy-borzenko**

### CODE REVIEW

File: src/app/components/lineage-graph/lineage-graph.component.ts [hunk 1]
```diff
@@ -38,12 +40,14 @@ export class LineageGraphComponent implements OnChanges, OnInit {
...
     public miniMapPosition: MiniMapPosition;
     public graphNodes: Node[];
+    public datasetKind: string;
 
     public readonly DEFAULT_AVATAR_URL = AppValues.DEFAULT_AVATAR_URL;
     public readonly LineageGraphNodeKind: typeof LineageGraphNodeKind = LineageGraphNodeKind;
     public readonly DatasetKind: typeof DatasetKind = DatasetKind;
 
     public ngOnInit(): void {
+        console.log("===>", this.currentDataset);
```
Comment (EvilAlexei) [inline src/app/components/lineage-graph/lineage-graph.component.ts:50]: I assume this is a leftover? :)
Please remove

File: src/styles.scss [hunk 2]
```diff
@@ -1517,6 +1517,81 @@ input {
...
+            background: #cdd2d9;
+        }
+
+        .badge-verified {
+            margin-left: 5px;
+            color: green;
+            background: #d9f8e2;
+        }
+    }
 }
```
Comment (EvilAlexei) [inline src/styles.scss:1597]: Why make these styles global?
I think it will be better if they will be for component - `lineage-graph.component.scss`

**Review by EvilAlexei — CHANGES_REQUESTED** [review]

File: src/styles.scss (hunk 2 above)
Comment (dmitriy-borzenko) [reply src/styles.scss:1597]: Ok

File: src/app/components/lineage-graph/lineage-graph.component.ts (hunk 1 above)
Comment (dmitriy-borzenko) [reply src/app/components/lineage-graph/lineage-graph.component.ts:50]: Removed

File: src/app/api/gql/fragments/fragment-dataset-details.graphql [hunk 3]
```diff
@@ -10,5 +10,8 @@ fragment DatasetDetails on Dataset {
       ...License
     }
     currentWatermark
+    currentDownstreamDependencies {
+      __typename
+    }
```
Comment (zaychenko-sergei) [inline src/app/api/gql/fragments/fragment-dataset-details.graphql:15]: Looks wrong

File: src/app/components/lineage-graph/lineage-graph.component.ts [hunk 4]
```diff
@@ -38,6 +41,7 @@ export class LineageGraphComponent implements OnChanges, OnInit {
     public showMiniMap = true;
     public miniMapPosition: MiniMapPosition;
     public graphNodes: Node[];
+    public datasetKind: string;
```
Comment (zaychenko-sergei) [inline src/app/components/lineage-graph/lineage-graph.component.ts:44]: enum.
If you need this to capitalize the letter, I guess it should be done as a pipe

File: src/app/components/lineage-graph/lineage-graph.component.spec.ts [hunk 5]
```diff
@@ -11,6 +11,11 @@ import { LineageGraphNodeData } from "src/app/dataset-view/additional-components
 import { TEST_AVATAR_URL } from "src/app/api/mock/auth.mock";
 import _ from "lodash";
 import AppValues from "src/app/common/app.values";
+import { MatIconModule } from "@angular/material/icon";
+import { DisplaySizeModule } from "src/app/common/pipes/display-size.module";
+import { DisplayTimeModule } from "../display-time/display-time.module";
+import { SharedTestModule } from "src/app/common/shared-test.module";
+import { mockGraphNode } from "src/app/dataset-view/additional-components/data-tabs.mock";
 
 describe("LineageGraphComponent", () => {
```
Comment (zaychenko-sergei) [inline src/app/components/lineage-graph/lineage-graph.component.spec.ts:20]: Missing a test that would check that the values of dataset properties are visible as parts of the panel. Basically, compare elements content with dataset attributes.

**Review by zaychenko-sergei — CHANGES_REQUESTED** [review]

File: src/app/components/lineage-graph/lineage-graph.component.ts (hunk 4 above)
Comment (dmitriy-borzenko) [reply src/app/components/lineage-graph/lineage-graph.component.ts:44]: Found pipe from Angular ==> titlecase

File: src/app/api/gql/fragments/fragment-dataset-details.graphql (hunk 3 above)
Comment (dmitriy-borzenko) [reply src/app/api/gql/fragments/fragment-dataset-details.graphql:15]: Deleted

File: src/app/components/lineage-graph/lineage-graph.component.html [hunk 6]
```diff
@@ -60,19 +65,26 @@
...
+                    [attr.x]="35"
+                    [attr.y]="18"
+                    [attr.font-weight]="'normal'"
+                >
                     {{ node.data.dataObject.accountName }}
                 </svg:text>
                 <svg:g [attr.transform]="'translate(' + node.dimension.width * 1.005 + ',0)'">
                     <!-- Temporary solution - in future icon will be choosen on explicit public/private attribute in the dataset -->
-                    <image
+                    <!-- <image
```
Comment (zaychenko-sergei) [inline src/app/components/lineage-graph/lineage-graph.component.html:81]: If it's garbage, clean it

File: src/app/components/lineage-graph/lineage-graph.component.ts [hunk 7]
```diff
@@ -42,11 +45,32 @@ export class LineageGraphComponent implements OnChanges, OnInit {
...
+    private checkWindowSize(): void {
+        this.changeLineageGraphView();
+    }
 
     public ngOnInit(): void {
         this.graphNodes = this.nodes;
     }
 
+    public changeLineageGraphView(): void {
+        const searchResultContainer: MaybeNull<HTMLElement> = document.getElementById("searchResultContainerContent");
```
Comment (zaychenko-sergei) [inline src/app/components/lineage-graph/lineage-graph.component.ts:63]: Can it really be null?
This creates an unreachable `else` branch. 
Look for `istanbul` directives I've placed in other similar situations, it at least cuts down the impossible statements/branches from consideration.

File: src/app/components/lineage-graph/lineage-graph.component.html (hunk 6 above)
Comment (dmitriy-borzenko) [reply src/app/components/lineage-graph/lineage-graph.component.html:81]: Removed

File: src/app/components/lineage-graph/lineage-graph.component.ts (hunk 7 above)
Comment (dmitriy-borzenko) [reply src/app/components/lineage-graph/lineage-graph.component.ts:63]: Use instead `ViewChild` directive

**Review by zaychenko-sergei — APPROVED** [review]

**Pull request merged**